File: two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        
        int [] answer = new int [2];
        int index = 0;
       
        
        for(int i = 0; i < nums.length; i++)
        {
            index++;
            for(int b = index; b < nums.length; b++)
            {
                if(nums[i] + nums[b] == target)
                {
                    answer[0] = i;
                    answer[1] = b;
                    return answer;
                }
            }
        }
        return answer;
        
    }
}

File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        numsMap = {}
        for i in range(len(nums)):
            if target - nums[i] in numsMap:
                return [i, numsMap[target - nums[i]]]
            numsMap[nums[i]] = i
        return []

        
         
                


                

